test/ffmpeg-qsv/vpp/overlay.py

```diff
@@ -0,0 +1,31 @@
+###
+### Copyright (C) 2023 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ....lib.ffmpeg.qsv.util import *
+from ....lib.ffmpeg.qsv.vpp import VppTest
+
+spec = load_test_spec("vpp", "overlay", "rgba")
```

Comment:
"rgba" seems unnecessary in this spec key... nothing about the test implementation explicitly requires it.  It is up to user test case config to supply their own `source1` which is not constrained to any particular format in the test even if other formats may not work or make sense.

Reply:
Maybe this will be needed in future, e.g. for a rgb (rgb0) case, the output format from ffmpeg sw decoder is gbrp format, we have to convert it to rgb0

Reply:
I will remove it in this spec. We may add it back if it is needed in future.

Reply:
removed, thx